packages/hash/frontend/src/blocks/page/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { Schema } from "prosemirror-model";
```

Comment:
I'm not against this if we plan to use it in other places, but if not maybe we could just keep this inside `PageTitle`?

Reply:
Yes, I don't remember I put this on a separate file tbh, let me put that into `PageTitle` back again :D

Reply:
Hmm, I did this do keep `PageTitle` shorter & easier to read. But putting this into a generic `utils.ts` does not make sense, right. So I moved `PageTitle` to it's own folder with own utils file.